backend/src/test/service/CachingService.test.ts

```diff
@@ -0,0 +1,41 @@
+import { CachingService } from '../../main/service/CachingService';
+import { CachingObject } from '../../main/data/CachingObject';
+
+describe('simple add/get for redis cache', function () {
+  // NOTE: RUNNING THESE TESTS WILL MODIFY THE REDIS SERVER!
+  let cachingService: CachingService;
+
+  beforeEach(() => {
+    cachingService = new CachingService();
+  });
+
+  afterEach(() => {
+    const client = cachingService.getClient();
+    client.quit();
+  });
+
+  test('adding lat/lon value', () => {
+    return cachingService
+      .cacheLatLonValues('ChIJIfBAsjeuEmsRdgu9Pl1Ps48', -34.2, -46.123)
+      .then((value: boolean) => {
+        expect(value).toBeTruthy();
+      })
+      .catch((e) => {
+        fail('Did not expect to fail: ' + e.message);
+      });
+  });
+
+  test('getting lat/lon value', async () => {
+    return cachingService
+      .getLatLonValue('ChIJIfBAsjeuEmsRdgu9Pl1Ps48')
+      .then((result: CachingObject) => {
+        expect(result).toHaveProperty('lat');
+        expect(result).toHaveProperty('lon');
+        expect(result.lat).toBe(-34.2);
+        expect(result.lon).toBe(-46.123);
```

Comment:
I think the elvis operator could cover the `toHaveProperty` part. Do you think this will work?
```suggestion
        expect(result?.lat).toBe(-34.2);
        expect(result?.lon).toBe(-46.123);
```

Reply:
Hmm I'm not sure if that'll work since it's going to check if result is null/undefined but I'm checking that result has properties lat/lon?

Reply:
Ah wait sorry I mean if `result.lat` doesn't exist, then it will be undefined. So then 

```
expect(result.lat).tobe(-34.2)
expect(undefined).tobe(-34.2)
// throws error
```

But doesn't matter now that I think about it 😅

Reply:
All good 😄 ! Does this look okay then?

Reply:
Yep looks great! Sorry for being so picky